Authentication/AuthenticationBootstrapper.swift

```diff
@@ -219,33 +235,29 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
 extension AuthenticationBootstrapper: LoginControllerTransitionDelegate {
     
     public func onSignup(controller: LoginController) {
```

Comment:
What does this `onSignup` means? When the user sign up or when he presses the button? If it is the second one, we should change its name